Add Game Jam Article (#1157)
huggingface/blog: game-jam.md

---
title: "Announcing our first AI Game Jam 🎮"
thumbnail: /blog/assets/145_gamejam/thumbnail.png
authors:
- user: ThomasSimonini
---

<h1>Announcing our first AI Game Jam 🎮 </h1>
<h2> Unleash Your Creativity with AI Tools and make a game in a weekend!</h2>

<!-- {blog_metadata} -->
<!-- {authors} --> 

We're thrilled to announce our **first-ever Open Source AI Game Jam**, where you will **create a game using cutting-edge AI tools**.

> With AI's potential to enhance game experiences and workflows, **we're excited to see what you can accomplish**: incorporate generative AI tools like Stable Diffusion into your game or workflow to unlock new features and accelerate your development process. 

Reviewer: maybe mention "like Stable Diffusion or LLMs" so that it doesn't look that it's only for visuals?

From texture generation to lifelike NPCs and realistic text-to-speech, the options are endless.

📆 Mark your calendars: the game jam will take place from Friday, **July 7th to Sunday, July 9th**.

> **Claim Your Free Spot in the Game Jam** 👉 https://itch.io/jam/open-source-ai-game-jam

Author: I think it's better to reinforce free

<h2>Why Are We Organizing This?</h2>

> In a time when some popular game jams restrict the use of AI tools, we believe it's crucial to **provide a platform specifically dedicated to showcasing the incredible possibilities AI offers game developers**.

Author: ✅

We want to see these jams thrive and empower indie developers with the tools they need to boost productivity and unlock their full potential.

<h2>What Are AI Tools?</h2>

AI tools, particularly generative ones like Stable Diffusion, open up a whole new world of possibilities in game development.

From accelerated workflows to in-game features, you can harness the power of AI for texture generation, lifelike AI non-player characters (NPCs), and realistic text-to-speech functionality.

Claim Your Free Spot in the Game Jam 👉 https://itch.io/jam/open-source-ai-game-jam

<h2>Who Can Participate?</h2>

> **Everyone is invited to join the Open Source AI Game Jam**, regardless of skill level or location.

Reviewer 2: ```suggestion
**Everyone is welcome to join the Open Source AI Game Jam**, regardless of skill level or location. You can participate alone or in a team of any size.
```

Whether you're a seasoned pro or just starting out, you can participate alone or form a team of any size.

<h2>What Are the Requirements?</h2>

To participate, your game should be playable on the web (e.g., itch.io) or Windows.
Additionally, **you are required to incorporate at least one open-source AI tool into your game or workflow**.

We'll provide more details to guide you along the way.

<h2>Can I Use Existing Assets?</h2>

Absolutely! **You're encouraged to use any existing assets, code, or AI tools that you have legal access to.**

We want to ensure fairness and give you the freedom to leverage the resources at your disposal.

<h2>Is There a Theme?</h2>

> Yes, the jam will **kick off with the announcement of a theme**. This adds an exciting twist to the creative process, encouraging participants to think outside the box and come up with unique game concepts.

Reviewer 2: ```suggestion
Yes, the theme will be announced when the jam starts.
```
Sounds too wordy and artificial IMO

<h2>How Will the Games Be Judged?</h2>

> During the Open Source AI Game Jam, participants **will have the opportunity to rate other games based on three criteria**: fun, creativity, and adherence to the theme.

Reviewer 2: ```suggestion
Participants will rate other games based on three criteria: **fun, creativity, and theme**. The judges will showcase and choose the winner from the Top 10.
```

From the top-rated games, our panel of judges will select the ultimate winner, considering the overall quality and innovation displayed.

<h2> Join our Discord Community! </h2>

> Want to connect with the community? Join our Discord to meet and exchange with our community of game developers and AI enthusiasts!

Reviewer 2: ```suggestion
Want to connect with the community? Join our Discord!
```
A bit verbose IMO

👉 https://discord.com/invite/hugging-face-879548962464493619

> **Claim Your Free Spot in the Game Jam** 👉 https://itch.io/jam/open-source-ai-game-jam

Reviewer 2: ```suggestion
**Claim Your Spot in the Game Jam** 👉 https://itch.io/jam/open-source-ai-game-jam
```
